app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/ImportationFilesManager.kt

```diff
@@ -190,9 +189,13 @@ class ImportationFilesManager @Inject constructor(
         return getColumnIndex(column).takeIf { it != -1 }
     }
 
-    private suspend fun reportFailedImportation(file: PickedFile) {
-        SentryLog.e(TAG, "Failed importation of ${file.uri}")
-        _failedFiles.emit(file)
+    private fun reportFailedImportation(file: PickedFile, throwable: Throwable) {
+        SentryLog.e(TAG, "Failed importation of ${file.uri}", throwable)
+
+        //TODO: Make more precise error messages (especially for the low storage issue).
+        val errorMessage = appContext.getString(R.string.cant_import_file_x, file.fileName)
+        @OptIn(UnreliableToastApi::class)
+        longToast(errorMessage) //TODO: Find a better way to show the error in an actionable way.
```

Comment:
When triggering the toast manually by throwing an error inside the copy process, I get this exception `java.lang.NullPointerException: Can't toast on a thread that has not called Looper.prepare()` at the moment of showing the toast

Reply:
Thanks for catching it, I made sure to have it run on the main thread.